### PR TITLE
Render messages on profile

### PR DESCRIPTION
**Pull request by joycezhao2**

### CODE REVIEW

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 1]
```diff
@@ -55,9 +57,6 @@
   /** All the conversations */
   private List <Conversation> conversations;
 
-  /* Only messages that correspond to a given user ID */
-  private List <Message> realMessages = new ArrayList<>();
```
Comment (swarelle) [inline src/main/java/codeu/controller/ProfileServlet.java]: Why was this previously an instance variable and why is it now moved into the doGet method as a local variable? Would it make things more efficient if we built an instance variable of a Map that maps Strings (usernames) to a List<Message> (that user's messages)?

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 2]
```diff
@@ -166,10 +167,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
...
-      userStore.getUser(userId).setAbout(about);
+      user.setAbout(about);
+      userStore.putUser(user);
       request.getSession().setAttribute("about", about);
     }
 
-    response.sendRedirect("/profile/" + userId.toString());
-  }
+    String url = "/profile/" + userId.toString();
+    response.sendRedirect(url);
```
Comment (swarelle) [inline src/main/java/codeu/controller/ProfileServlet.java]: Why separate the original line into two?

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 1 above)
Comment (joycezhao2) [reply src/main/java/codeu/controller/ProfileServlet.java]: I noticed when it was an instance variable each time the page was rendered, more messages were just added to realMessages, ending up in a bunch of repeated messages being displayed on the profile. 

Making it a local variable makes sure that each message is only rendered once I think!

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 2 above)
Comment (joycezhao2) [reply src/main/java/codeu/controller/ProfileServlet.java]: It was just something leftover from debugging!

**Pull request merged**